FeedStoreChallenge/CoreDataFeedStore/CoreDataFeedStore.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  CoreDataFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Saba Khutsishvili on 3/6/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+public class CoreDataFeedStore: FeedStore {
+	private let container: NSPersistentContainer
+	private let context: NSManagedObjectContext
+	
+	public init(storeURL: URL, bundle: Bundle = .main) throws {
+		container = try NSPersistentContainer.load(
+			named: "\(CoreDataFeedStore.self)",
```

Comment:
Ideally, you should use the model name as a hardcoded string here. Otherwise, if someone renames this class or the data model, the code will break because it won't match the data model name anymore.

```suggestion
			named: "CoreDataFeedStore",
```